pkg/controller/v1beta1/inferenceservice/reconcilers/raw/raw_reconciler.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2021 kubeflow.org.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package raw
+
+import (
+	"github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1"
+	deployment "github.com/kubeflow/kfserving/pkg/controller/v1beta1/inferenceservice/reconcilers/deployment"
+	ingress "github.com/kubeflow/kfserving/pkg/controller/v1beta1/inferenceservice/reconcilers/ingress"
+	service "github.com/kubeflow/kfserving/pkg/controller/v1beta1/inferenceservice/reconcilers/service"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	knapis "knative.dev/pkg/apis"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//NativeReconciler is the struct of Native K8S Object
+type RawReconciler struct {
+	client          client.Client
+	scheme          *runtime.Scheme
+	Deployment      *deployment.DeploymentReconciler
```

Comment:
lower case as this does not need to be exposed and can be named as `deploymentReconciler`